tests/plugins/sort.test.ts

```diff
@@ -337,6 +337,41 @@ describe("Basic Sorting", () => {
       A7: { content: "-33" },
     });
   });
+
+  test("Sort with a cell that will be removed because it is considered empty", () => {
+    model = new Model({
+      sheets: [
+        {
+          id: sheetId,
+          colNumber: 1,
+          rowNumber: 3,
+          cells: {
+            A1: { content: "a" },
+            A2: { content: '=""' },
+          },
+        },
+      ],
+    });
+    model.dispatch("SORT_CELLS", {
+      sheetId,
+      col: 0,
+      row: 1,
+      zone: {
+        top: 1,
+        left: 0,
+        bottom: 2,
+        right: 0,
+      },
+      sortDirection: "ascending",
+      sortOptions: {
+        emptyCellAsZero: true,
+        sortHeaders: true,
+      },
+    });
```

Comment:
You can use the test helper :wink:  It makes the test much easier to read with the range references as strings
```ts
    sort(model, {
      zone: "A2:A3",
      anchor: "A2",
      direction: "ascending",
      sortOptions: { emptyCellAsZero: true },
    });
```
Also, `sortHeaders` isn't relevent for this test, it can be omitted.

Reply:
Sure, thank you ! I was actually hoping you would correct me here. I was not able to find the right combination of parameters with the test helper 😅